Description: Don't package several bitmaps of unknown copyright
 The removals will happen automatically as those files are not included in the new automake-based build system.
 This patch makes substitutes for the two that were specifically used.
Author: David Hart (david@4Pane.co.uk)
diff --git a/Configure.cpp b/Configure.cpp
index bf612c9..680ba09 100644
--- a/Configure.cpp
+++ b/Configure.cpp
@@ -933,22 +933,20 @@ else
 wxConfigBase::Get()->Write(wxT("Editors/Icons/d/Icontype"), 15l);
 wxConfigBase::Get()->Write(wxT("Editors/Icons/e/Icon"),  wxT("chrome-chromium.png"));
 wxConfigBase::Get()->Write(wxT("Editors/Icons/e/Icontype"), 15l);
-wxConfigBase::Get()->Write(wxT("Editors/Icons/f/Icon"),  wxT("editor.xpm"));
-wxConfigBase::Get()->Write(wxT("Editors/Icons/f/Icontype"), 9l);
-wxConfigBase::Get()->Write(wxT("Editors/Icons/g/Icon"),  wxT("mousepad.png"));
+wxConfigBase::Get()->Write(wxT("Editors/Icons/f/Icon"),  wxT("mousepad.png"));
+wxConfigBase::Get()->Write(wxT("Editors/Icons/f/Icontype"), 15l);
+wxConfigBase::Get()->Write(wxT("Editors/Icons/g/Icon"),  wxT("openoffice.png"));
 wxConfigBase::Get()->Write(wxT("Editors/Icons/g/Icontype"), 15l);
-wxConfigBase::Get()->Write(wxT("Editors/Icons/h/Icon"),  wxT("openoffice.png"));
+wxConfigBase::Get()->Write(wxT("Editors/Icons/h/Icon"),  wxT("libreoffice.png"));
 wxConfigBase::Get()->Write(wxT("Editors/Icons/h/Icontype"), 15l);
-wxConfigBase::Get()->Write(wxT("Editors/Icons/i/Icon"),  wxT("libreoffice.png"));
+wxConfigBase::Get()->Write(wxT("Editors/Icons/i/Icon"),  wxT("mate-text-editor.png"));
 wxConfigBase::Get()->Write(wxT("Editors/Icons/i/Icontype"), 15l);
-wxConfigBase::Get()->Write(wxT("Editors/Icons/j/Icon"),  wxT("mate-text-editor.png"));
+wxConfigBase::Get()->Write(wxT("Editors/Icons/j/Icon"),  wxT("gjots.png"));
 wxConfigBase::Get()->Write(wxT("Editors/Icons/j/Icontype"), 15l);
-wxConfigBase::Get()->Write(wxT("Editors/Icons/k/Icon"),  wxT("gjots.png"));
-wxConfigBase::Get()->Write(wxT("Editors/Icons/k/Icontype"), 15l);
-wxConfigBase::Get()->Write(wxT("Editors/Icons/l/Icon"),  wxT("unknown.xpm"));
-wxConfigBase::Get()->Write(wxT("Editors/Icons/l/Icontype"), 9l);
-wxConfigBase::Get()->Write(wxT("Editors/Icons/m/Icon"),  wxT("palemoon.png"));
-wxConfigBase::Get()->Write(wxT("Editors/Icons/m/Icontype"), 15l);
+wxConfigBase::Get()->Write(wxT("Editors/Icons/k/Icon"),  wxT("unknown.xpm"));
+wxConfigBase::Get()->Write(wxT("Editors/Icons/k/Icontype"), 9l);
+wxConfigBase::Get()->Write(wxT("Editors/Icons/l/Icon"),  wxT("palemoon.png"));
+wxConfigBase::Get()->Write(wxT("Editors/Icons/l/Icontype"), 15l);
 
 wxConfigBase::Get()->Write(wxT("/Editors/NoOfEditors"), 0l);    // Save a zero count first, just so that it's stored first (& so can be more easily seen)
 long count=0; wxString path(wxT("/Editors/"));
diff --git a/MyGenericDirCtrl.cpp b/MyGenericDirCtrl.cpp
index 847160e..20deb0f 100644
--- a/MyGenericDirCtrl.cpp
+++ b/MyGenericDirCtrl.cpp
@@ -293,7 +293,7 @@ static const char * icon8_xpm[] = {
 #include "bitmaps/include/pipe.xpm"                                   // // Icon for FIFO
 #include "bitmaps/include/blockdevice.xpm"                            // // Icon for Block device
 #include "bitmaps/include/chardevice.xpm"                             // // Icon for Char device
-#include "bitmaps/include/socket.xpm"                                 // // Icon for Socket
+#include "bitmaps/connect_no.xpm"                                     // // Icon for Socket
 #include "bitmaps/include/compressedfile.xpm"                         // // Icon for compressed files
 #include "bitmaps/include/tarball.xpm"                                // // Icon for tarballs et al e.g. .a 
 #include "bitmaps/include/compressedtar.xpm"                          // // Icon for compressed tarballs, .cpio, packages e.g. rpms
@@ -478,7 +478,7 @@ if (!USE_DEFAULT_TREE_FONT && CHOSEN_TREE_FONT.Ok())  SetFont(CHOSEN_TREE_FONT);
     m_imageList->Add(wxIcon(pipe_xpm));                           // //
     m_imageList->Add(wxIcon(blockdevice_xpm));                    // //
     m_imageList->Add(wxIcon(chardevice_xpm));                     // //
-    m_imageList->Add(wxIcon(socket_xpm));                         // //
+    m_imageList->Add(wxIcon(connect_no_xpm));                     // //
     m_imageList->Add(wxIcon(compressedfile_xpm));                 // //
     m_imageList->Add(wxIcon(tarball_xpm));                        // //
     m_imageList->Add(wxIcon(compressedtar_xpm));                  // //
